blob/fileblob/fileblob.go

```diff
@@ -148,6 +152,7 @@ func (r reader) Attrs() *driver.ObjectAttrs {
 	return &driver.ObjectAttrs{
 		Size:        r.size,
 		ContentType: r.xa.ContentType,
+		ModTime:     r.modTime,
```

Comment:
Please also add tests for this new functionality for `fileblob`.

Reply:
Add test in [7e530ee](https://github.com/google/go-cloud/pull/240/commits/7e530ee85c8ebea2c231011465279c1042b402a4)